currency_converter.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import sys
+
+from forex_python.converter import CurrencyCodes
+from forex_python.converter import CurrencyRates
+from forex_python.converter import RatesNotAvailableError
+
+
+code_dictionary = \
```

Comment:
The `\` is there so I can put `{` on next line of course. We usually use this formatting in haskell in current job, although there is no need for `\` in haskell.

It's just current personal choice, but I usually follow project's coding style when I contribute somewhere.